docs/weaviate/concepts/vector-quantization.md

```diff
@@ -145,6 +137,13 @@ This means that the feature is still under development and may change in future
 
 ### 1-bit RQ
 
+:::info Added in `v1.33` and `v1.34`
+
+**1-bit Rotational quantization (RQ)** for the HNSW vector index was added in **`v1.33`**.<br/>
+**1-bit Rotational quantization (RQ)** for the flat vector index was added in **`v1.34`** as a **preview** and should not be used in production.<br/>
```

Comment:
Given this is a warning not to use, imo this could be a separate :::caution::: admonition imho.